rest-api-cloud-client/src/main/java/com/stnetix/ariaddna/externalcloudapi/cloudinterface/iAbstractCloud.java

```diff
@@ -0,0 +1,100 @@
+package com.stnetix.ariaddna.externalcloudapi.cloudinterface;
+
+/**
+ * This is an abstract interface containing general methods for
+ * manipulation with resources at any external cloud storage.
+ * Every certain cloud storage realization must implement this interface.
+ */
+public interface iAbstractCloud {
+    /**
+     * An uploadResource method implementation takes as a parameter a path to a resource
+     * which is stored at user's local HDD/SSD and sends it to an external cloud storage.
+     * Notice, that resource may be a file or directory with files.
+     *
+     * @param path A path to a resource at user's local storage.
+     * @return String representation of response with usefull
+     * information which may be converted into JSON Object.
+     */
+
+    String uploadResource(String path);
```

Comment:
If you have resource type is file or directory, may be use parameter as Java type File?

What is return vales of String type? What contain this String?